site/src/Controller/OrganizationController.php

```diff
@@ -215,6 +220,20 @@ public function joinAction($organizationName, OrganizationRepository $repository
         $member->setUser($user);
         $member->setPermission(OrganizationMember::PERMISSION_REQUEST);
         $em = $this->getDoctrine()->getManager();
+        // Find Organisation members
+        $orgAdmin = $repository->findOneByName($organizationName)->getCreator();
```

Comment:
Please send this notification to all admins (can be more than one). They are OrganizationMember entities with the PERMISSION_ADMIN permission set.